smart_pet_buddy/lib/signin_page.dart

```diff
@@ -131,8 +104,9 @@ class _EmailPasswordFormState extends State<_EmailPasswordForm> {
                 Container(
                   padding: const EdgeInsets.only(top: 16),
                   alignment: Alignment.center,
-                  child: SignInButton(
-                    Buttons.Email,
+                  child: SignInButtonBuilder(
```

Comment:
Nice, did the SignInButton cause any restrictions in the design?

Reply:
Yes exactly! Had to change in order to be able to choose a different color of the button for example.

Reply:
Good job!